### DISCUSSION TITLE
What kind of VU , dB ?

### DISCUSSION THREAD

**Discussion by pcar75** [jamulussoftware/jamulus#3295] [Technical Support & Help] [answered]
Bonjour,

On reading about volume levels (Sound On Sound magazine 2024May/"Enjoy the VU" and 2003October/Q&A/"Are all Decibels equal?"  and various web entries) I am curious about what exactly do the tracks bar-graph levels represent.  When playing guitar, I aim to stay about the 2 yellow bars of my Input track.  I think this is Ok but what level does that correspond to ? (Note that I'm no sound engineer !)

Thanks for any info !

**Comment by ann0see** [comment]
This won't answer your question fully, but see https://github.com/jamulussoftware/jamulus/blob/f9655187e99b6192c9bb4bd153d86ca57b788c79/src/global.h#L199-L206 where the levels are defined and https://github.com/jamulussoftware/jamulus/blob/f9655187e99b6192c9bb4bd153d86ca57b788c79/src/levelmeter.cpp#L194 where the change to yellow happens.

In end effect, I'd just trust what others and my ears say.

**Reply by pcar75** [reply, marked as answer]
Thanks, So do I.  But it's just out of curiosity ...
So the comments on the code would indicate -50 to 0 dB in 9 segments (0-8 , given C/C++ indexing )
of approximately 11.1%  each.  Linearly that would use 5,6 dB per segments, however the scaling could be logarithmic.
From some other sources, these would be dBFS or dB Full Scale I suppose, as used in most audio DAW.
